src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -216,6 +222,11 @@
     </Compile>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
```

Comment:
We have the ids defined at https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#specific-platform-mapping but we probably should still put more information in our project guidelines as well.